utils/deployer/truffle-config.js

```diff
@@ -0,0 +1,52 @@
+require('dotenv').config()
+const HDWalletProvider = require('@truffle/hdwallet-provider')
+const HDWalletProviderPriv = require('truffle-hdwallet-provider-privkey')
+
+module.exports = {
+  contracts_directory: './flatten',
+  networks: {
+    development: {
+      host: '127.0.0.1', // Localhost (default: none)
```

Comment:
not sure what you mean by default: none?

Reply:
this is automatically generated by truffle. will remove.